docs/gettingstarted/mk8s/k8s-microk8s-cluster.md

```diff
@@ -0,0 +1,74 @@
+# Clustering SNMP with microk8s
```

Comment:
maybe we could have here like 1-2 sentences about what this instruction is about.